VTKConverter/VTKConverter/ModelConversion/ModelConverter.cs

```diff
@@ -23,22 +25,22 @@ private void CreateOutputRoot(string inputRootDir, string outputFolder)
 
         private void ConvertSingleModel(SingleModel singleModel)
         {
-            Console.WriteLine("Conversion started!");
+            Log.Info("Conversion started!");

```

Comment:
How about additional info about model in this log?

Reply:
Added model's name to the string.